lexicon.txt

```diff
@@ -0,0 +1,1317 @@
+aaaa
+aaaaaaa
+aal
+abuflen
+ack
+acked
+ackno
+acks
+acount
+acs
+addr
+addressalignedbeats
+addrstart
+addtogroup
+advertize
+afe
+ahb
+amba
+ap
+api
+apis
+april
+arg
+arp
+arppacket
+arpprocesspacket
+arprefreshcacheentry
+arps
+ascii
+asf
+atmel
+automaticpadcrcstrip
+autonegotiation
+aws
+backoff
+backofflimit
+backpressure
+bactive
+bafter
+barry
+baseaddress
+bbbbbbb
+bbefore
+bcloseaftersend
+bcloserequested
+bconnpassed
+bconnprepared
+bds
+berkeley
+besr
+bexpired
+bfinaccepted
+bfinacked
+bfinlast
+bfinrecv
+bfinsent
+bhasinit
+bisforrx
+bist
+bitmask
+bitscanreverse
+bl
+blowwater
+bmallocerror
+bmayconnect
+bmsschange
+bool
+boolean
+boutstanding
+bpassaccept
+bpassqueued
+bpdg
+breleaseaftersend
+breusesocket
+broadcastframesreception
+brxstopped
+bsd
+bsendfullsize
+bsendkeepalive
+btimestamps
+buf
+buffermanagement
+buffna
+busershutdown
+bwaitkeepalive
+bwinchange
+bwinscaling
+byt
```

Comment:
this looks like a misspelling of _byte_

Reply:
It was indeed! Thanks for catching that! Updated.